src/main/java/com/epam/cube/logic/CubeCalculator.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.cube.logic;
+
+import com.epam.cube.entity.Axes;
+import com.epam.cube.entity.Cube;
+import com.epam.cube.entity.Point;
+
+public class CubeCalculator {
+
+    public boolean isCube(Point startingPoint, double side){
```

Comment:
could be moved to CubeValidator as it performs validation, also this method is used only in tests